src/main/java/fr/insee/genesis/controller/utils/LoopIdentifier.java

```diff
@@ -15,6 +18,18 @@ private LoopIdentifier() {
 	public static String getLoopIdentifier(String variableName, VariablesMap variablesMap, int index) {
 		Variable variable = variablesMap.getVariable(variableName);
 		if (variable == null) {
+			if(variableName.startsWith(Constants.FILTER_RESULT_PREFIX)
+					&& variablesMap.hasVariable(variableName.replace(Constants.FILTER_RESULT_PREFIX,""))
+			){
+				Variable parentVariable = variablesMap.getVariable(variableName.replace(Constants.FILTER_RESULT_PREFIX,""));
+				return parentVariable.getGroupName();
+			}
+			if(variableName.endsWith(Constants.MISSING_SUFFIX)
```

Comment:
this "if" is exactly the same as above. Maybe create a method "getGroupNameForSpecificVariable(String constant to replace, beginOrEnd, variableName) ?